build.gradle

```diff
@@ -75,7 +75,7 @@ subprojects {
                 'neo4jCommunityDockerImage': System.getProperty("NEO4JVERSION") ? 'neo4j:' + System.getProperty("NEO4JVERSION") : 'neo4j:5.6.0',
                 'coreDir': 'core'
 
-        maxHeapSize = "8G"
+        maxHeapSize = "5G"
```

Comment:
I'm not sure this had any impact, but GitHub Actions machines have 6GB of memory, so if this was honoured it could be problematic

Reply:
512m to run the tests is too little. The settings in 4.4 should be changed, not these ones

Reply:
Ah now I get your point. Don't you think anyway having a consistent setup for local and CI runs (like having the same heap space) could be better?

Reply:
Yeah well, thinking about it maybe it's not the best to have 2 different setups.
That's okay then :)